### PR TITLE
Vomit observation entry fix

### PR DESCRIPTION
**Pull request by kollil**
Added vomit entry as an observation

### CODE REVIEW

**Review by Ohsudev — APPROVED** [review]

**Comment by kollil** [comment]
@labkey-martyp
The automated tests are failing on this update. It's saying cannot find the "vomit" query in ehr_lookups. 
I have added all the necessary data that's needed to populate the vomit data as an observation. 
Could you please check and suggest if I am missing anything?

Thanks
Lakshmi

File: onprc_ehr/resources/data/lookup_sets.tsv [hunk 1]
```diff
@@ -43,6 +43,7 @@ alopecia_type	Alopecia Type	value
 alopecia_score	Alopecia Type	value
 alopecia_regrowth	Alopecia Regrowth	value
 mens_score	Mens Score	value
+vomit   Vomit   value
```
Comment (labkey-martyp) [inline onprc_ehr/resources/data/lookup_sets.tsv]: The whitespace between these values needs to be tabs. You have spaces here.

Comment (kollil) [reply onprc_ehr/resources/data/lookup_sets.tsv]: @labkey-martyp, I have made that correction and automated tests still failed with the same errors.

Comment (jryurkanin) [reply onprc_ehr/resources/data/lookup_sets.tsv]: @kollil  Marty's requested changes were not implemented although it looks like you found another place where tabs should be used. Marty was talking abou thte lookup_sets.tsv file. To demonstrate the issue, download the tsv and open it in excel. https://github.com/LabKey/onprcEHRModules/blob/23.11_fb_VomitObs/onprc_ehr/resources/data/lookup_sets.tsv

Please notice that this comment thread start by Marty is pointing at the lookup_sets.tsv file, not the vomit.tsv file.

Comment (kollil) [reply onprc_ehr/resources/data/lookup_sets.tsv]: @jryurkanin, Thanks for giving me the tip of downloading into an excel. I did add the tabs in the code first time Marty pointed out, but, somehow the tabs were not getting inserted in the code. I copied the entire tsv file into wordpad, added tabs and then pasted back into the code. Finally it worked.

Comment (jryurkanin) [reply onprc_ehr/resources/data/lookup_sets.tsv]: Glad that worked for you. I actually recommend building the TSV files in excel, then you can copy and paste the table from excel straight into the github website code editor. When you do that, it retains the correct tabular format from excel.

**Pull request merged**